### PR TITLE
[tr] Linear algebra

### PR DESCRIPTION
**Pull request by ktekeli**

### CODE REVIEW

**Comment by shervinea** [comment]
Thanks for your great work @ktekeli!

Now, let's wait for reviewers to go through this translation. (cc: @ekremcet)

File: tr/refresher-linear-algebra.md [hunk 1]
```diff
@@ -1,339 +1,339 @@
...
 
-&#10230;
+&#10230; [Matris özellikleri, Norm, Özdeğer/Özvektör, Tekil-değer ayrışımı]
 
 <br>
 
 **57. [Matrix calculus, Gradient, Hessian, Operations]**
 
-&#10230;
+&#10230; [Matris kalkülüsü, Gradiyent, Hessian, İşlemler]
```
Comment (ekremcet) [inline tr/refresher-linear-algebra.md]: Maybe Gradient can be translated as "Gradyan"

File: tr/refresher-linear-algebra.md [hunk 2]
```diff
@@ -1,339 +1,339 @@
...
 
-&#10230;
+&#10230; Uyarı: f fonksiyonunun Hessian’ı yalnızca f skaler döndüren bir fonksiyon ise tanımlıdır.
 
 <br>
 
 **53. Gradient operations ― For matrices A,B,C, the following gradient properties are worth having in mind:**
 
-&#10230;
+&#10230; Gradiyent işlemleri ― A,B,C matrisleri için aşağıdaki işlemlerin akılda bulunmasında fayda vardır:
```
Comment (ekremcet) [inline tr/refresher-linear-algebra.md]: Same as Line 339

File: tr/refresher-linear-algebra.md [hunk 3]
```diff
@@ -1,339 +1,339 @@
...
 
-&#10230;
+&#10230; Matris kalkülüsü
 
 <br>
 
 **49. Gradient ― Let f:Rm×n→R be a function and A∈Rm×n be a matrix. The gradient of f with respect to A is a m×n matrix, noted ∇Af(A), such that:**
 
-&#10230;
+&#10230; Gradiyent ― f:Rm×n→R bir fonksiyon ve A∈Rm×n bir matris olsun. f nin A ya göre gradiyenti m×n tipinde bir matristir, ve ∇Af(A) ile gösterilir:
```
Comment (ekremcet) [inline tr/refresher-linear-algebra.md]: Same as Line 339

File: tr/refresher-linear-algebra.md [hunk 4]
```diff
@@ -1,339 +1,339 @@
...
 
-&#10230;
+&#10230; Birim matris ― Birim matris, köşegeni birlerden ve diğer tüm elemanları sıfırlardan oluşan karesel matris, I∈Rn×n:
 
 <br>
 
 **9. Remark: for all matrices A∈Rn×n, we have A×I=I×A=A.**
 
-&#10230;
+&#10230; Uyarı: Her A∈Rn×n matrisi için A×I=I×A=A eşitliği sağlanır.
```
Comment (ekremcet) [inline tr/refresher-linear-algebra.md]: I think "Dipnot" or "Hatırlatma" is more appropriate translation for "Remark" here

**Comment by ktekeli** [comment]
@ekremcet thanks for your review and suggestions.

**Comment by shervinea** [comment]
Thanks a lot to the both of you @ktekeli and @ekremcet. I am moving forward with the merge!

**Pull request merged**